langserver/util.go

```diff
@@ -31,3 +32,44 @@ func pathEqual(a, b string) bool {
 func IsVendorDir(dir string) bool {
 	return strings.HasPrefix(dir, "vendor/") || strings.Contains(dir, "/vendor/")
 }
+
+// isUri tells if s denotes an URI
+func isUri(s string) bool {
+	return strings.HasPrefix(s, "file://")
+}
+
+// pathToUri converts given path to file URI
+func pathToUri(path string) string {
+	prefix := "file://"
+	if !strings.HasPrefix(path, "/") {
+		// On Windows there should be triple slash
+		prefix += "/"
+	}
+	path = strings.Replace(path, "\\", "/", -1)
```

Comment:
@keegancsmith, @felixfbecker addressed your remarks. If path looks like Windows-one (second rune is `:`), converting backslashes to slashes before converting it to URI, otherwise leaving it as is, url-encoding backslashes inside.